src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,125 @@
+$card-width: 200px;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$main-button-color: #00acdc;
+$main-title-hover-color: #34568b;
+$main-background-color: #fff;
+$main-border-color: #f3f3f3;
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  height: 408px;
```

Comment:
Don't fix container size (if there is no such a requirement). Fixed width make sense, but not height